packages/shared/src/components/modals/common.tsx

```diff
@@ -4,3 +4,7 @@ export const ModalHeader = classed(
   'header',
   'flex flex-row items-center py-2 px-3 h-14 border-b border-theme-divider-tertiary',
```

Comment:
Not sure if this is part of the new guideline - but the padding for the header is now `24px` = `px-6`. If so, then should we make small updates from here or do it in a single separate PR?
```suggestion
  'flex flex-row items-center py-3 px-3 h-14 border-b border-theme-divider-tertiary',
```

Reply:
Let's refactor right away, it should reflect all headers anyway.
I will add another ticket to revisit modals as they don't all use this sytem.